src/taipy/core/data/_data_repository.py

```diff
@@ -112,7 +112,8 @@ def _to_model(self, data_node: DataNode):
             data_node.owner_id,
             list(data_node._parent_ids),
             data_node._last_edit_date.isoformat() if data_node._last_edit_date else None,
-            data_node._job_ids.data,
+            [],  # DEPRECATED _job_ids (using _edits now)
+            data_node._edits,
```

Comment:
We can remove it here as well.

Reply:
I don't think so: we need to keep it for legacy reasons.
Removing it everywhere would break the compatibility.